cfme/tests/v2v/test_csv_import.py

```diff
@@ -296,12 +295,15 @@ def test_csv_security_group_flavor(appliance, infra_map, valid_vm, provider):
         flavor = provider.data.flavors[1]
     except (AttributeError, KeyError):
         pytest.skip("No provider data found.")
-    content = "Name,Security Group,Flavor\n{valid_vm},{security_group},{flavor}".format(
+    content = "Name,Security Group,Flavor\n{valid_vm},{security_group},{flavor}\n".format(
         valid_vm=valid_vm,
         security_group=security_group,
         flavor=flavor,
     )
-    actual_attributes = {"security_group": security_group, "flavor": flavor}
-    expected_attributes = check_vm_status(appliance, infra_map, actual_attributes, content=content,
+
+    expected_attributes = check_vm_status(appliance, infra_map, content=content,
                                           table_hover=True, security_group=True)
-    assert not DeepDiff(actual_attributes, expected_attributes)
+
+    soft_assert(expected_attributes["security_group"] == security_group)
```

Comment:
Why are you using soft_assert here instead of assert?
From what I understand, you only use soft_assert when you have to assert more than 1 statement in a single test to pass and you don't want your test to break in the middle if any one of the assert statements fails.

Reply:
down the line we have one more assert which is why i have used soft_assert above.